### ISSUE TITLE
Command line interface

### ISSUE THREAD

**Issue by wokalski** [wokalski/gobi#9]
What options should gobi have?

**Comment by tekknolagi** [comment]
OCaml compiler, toolchain directories, maybe everything after https://github.com/wokalski/gobi/blob/master/gobi#L16 ?

**Comment by wokalski** [comment]
I don't think I understand what you mean. I feel like it's the right choice that it builds for all architectures at once (and that's what this script does). I'm wondering about `ocamlopt` options, linking libraries - things I have no idea about.

**Comment by tekknolagi** [comment]
Oh, I see what you mean. I figured it could be useful to compile for a particular target architecture if desired. Unclear on the rest.

**Comment by wokalski** [comment]
You are right in the sense that it's 4 times as much work to be done, but I don't expect building the libs to occur so often that it becomes annoying. It will definitely be desired if/when `gobi` builds executables, not a static library (#3).
